Packages/vcs/Lib/vtk_ui/button.py

```diff
@@ -95,6 +95,8 @@ def __init__(self, interactor, action=None, corner_radius=5, width=None, font="A
         super(Button, self).__init__(interactor, widget)
 
         if tooltip:
+            if tooltip_property is not None:
+                tooltip_property.SetVerticalJustificationToTop()
```

Comment:
The tooltip's vertical justification was defaulting to centered, but I was expecting it to be Top.